contrib/for-tests/entrypoint-tester/Makefile

```diff
@@ -0,0 +1,18 @@
+all: push
+
+TAG = 0.1
+
+ep: ep.go
+	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags '-w' ./ep.go
+
+cmd: cmd.go
+	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags '-w' ./cmd.go
+
+image: ep cmd
+	sudo docker build -t kubernetes/eptest:$(TAG) .
+
+push: container
```

Comment:
I did mean 'image', thanks
On Mon, Mar 30, 2015 at 8:45 PM Victor Marmol <notifications@github.com>
wrote:

> In contrib/for-tests/entrypoint-tester/Makefile
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6100#discussion_r27445344>
> :
>
> > @@ -0,0 +1,18 @@
> > +all: push
> > +
> > +TAG = 0.1
> > +
> > +ep: ep.go
> > +	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags '-w' ./ep.go
> > +
> > +cmd: cmd.go
> > +	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags '-w' ./cmd.go
> > +
> > +image: ep cmd
> > +	sudo docker build -t kubernetes/eptest:$(TAG) .
> > +
> > +push: container
>
> do you mean image instead of container?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6100/files#r27445344>
> .
>